tsconfig.json

```diff
@@ -17,7 +17,8 @@
     "baseUrl": ".",
     "paths": {
       "@components": ["components/index"],
-      "@components/*": ["components/*"]
+      "@components/*": ["components/*"],
+      "types/*": ["types/*"]
```

Comment:
Added TypeScript alias which also translates to a webpack alias since this is a NEXT.js project.